CodeMaid/Helpers/CodeItemTypeComparer.cs

```diff
@@ -1,7 +1,7 @@
+using System.Collections.Generic;
```

Comment:
Nitpick, but can you sort the using statements alphabetically?  This is the default as of VS2012, even though previously System always came first and some tools like ReSharper still enforce it.